posters-overrides.json

```
@@ -3873,7 +3873,9 @@
    "number": 1370
   },
   {
-   "number": 1371
+   "number": 1371,
+   "title": "Multivariate analysis of atypical patterns of fMRI connectivity modulation in ASD",   
+   "pdf": "https://github.com/tlooden/posters/blob/master/OHBM_2020_poster_triloo_v3.pdf"   
```

```suggestion
   "pdf": "https://github.com/tlooden/posters/raw/master/OHBM_2020_poster_triloo_v3.pdf"
```
so could be just downloaded